### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by joeldodge79**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@looker/extension-sdk: 21.4.2</summary>

### Features

* Add clipboard write capability to extension framework ([#603](https://www.github.com/looker-open-source/sdk-codegen/issues/603)) ([d6b52ea](https://www.github.com/looker-open-source/sdk-codegen/commit/d6b52ea4f009f64e43167a4acac973fcbfb91e2e))
</details>
<details><summary>@looker/sdk-node: 21.4.2</summary>

### Bug Fixes

* error handling for TS SDK, GSon body serialization for Kotlin SDK ([#601](https://www.github.com/looker-open-source/sdk-codegen/issues/601)) ([11e924f](https://www.github.com/looker-open-source/sdk-codegen/commit/11e924fb81af00c7b20a1af2330f43873c016f83)), closes [#544](https://www.github.com/looker-open-source/sdk-codegen/issues/544)
</details>
<details><summary>sdk-codegen-all: 1.3.0</summary>

### Features

* Add clipboard write capability to extension framework ([#603](https://www.github.com/looker-open-source/sdk-codegen/issues/603)) ([d6b52ea](https://www.github.com/looker-open-source/sdk-codegen/commit/d6b52ea4f009f64e43167a4acac973fcbfb91e2e))
* adds 'SDK Examples' data table ([#602](https://www.github.com/looker-open-source/sdk-codegen/issues/602)) ([3678c96](https://www.github.com/looker-open-source/sdk-codegen/commit/3678c96cc7c2edadd00013b66711c917a62d1000))
* introduces prism-editor package; implements read-only usage ([#588](https://www.github.com/looker-open-source/sdk-codegen/issues/588)) ([863c88d](https://www.github.com/looker-open-source/sdk-codegen/commit/863c88dfb0569f7c829602acee2404e1c840097e))
* Playground for extension SDK changes ([#605](https://www.github.com/looker-open-source/sdk-codegen/issues/605)) ([f99de7e](https://www.github.com/looker-open-source/sdk-codegen/commit/f99de7e5bf590056b797b4b05c7a0a1c60f9d8dd))

### Bug Fixes

* error handling for TS SDK, GSon body serialization for Kotlin SDK ([#601](https://www.github.com/looker-open-source/sdk-codegen/issues/601)) ([11e924f](https://www.github.com/looker-open-source/sdk-codegen/commit/11e924fb81af00c7b20a1af2330f43873c016f83)), closes [#544](https://www.github.com/looker-open-source/sdk-codegen/issues/544)
* lint dependencies ([#617](https://www.github.com/looker-open-source/sdk-codegen/issues/617)) ([7b172b1](https://www.github.com/looker-open-source/sdk-codegen/commit/7b172b1c6927fbf62eb668a503313ced10558dfd))
* looker component version merge issue ([#609](https://www.github.com/looker-open-source/sdk-codegen/issues/609)) ([6aa9f60](https://www.github.com/looker-open-source/sdk-codegen/commit/6aa9f601019bb3f7105c48e3d360c3e61f57c6e1))
* updated TypeScript, C#, Go language names in codeGenerators.ts ([#610](https://www.github.com/looker-open-source/sdk-codegen/issues/610)) ([13fb42b](https://www.github.com/looker-open-source/sdk-codegen/commit/13fb42b6bfa651a137e75b04fc6ee00620705ede))
</details>
<details><summary>@looker/api-explorer: 0.9.3</summary>

### Features

* adds 'SDK Examples' data table ([#602](https://www.github.com/looker-open-source/sdk-codegen/issues/602)) ([3678c96](https://www.github.com/looker-open-source/sdk-codegen/commit/3678c96cc7c2edadd00013b66711c917a62d1000))
* introduces prism-editor package; implements read-only usage ([#588](https://www.github.com/looker-open-source/sdk-codegen/issues/588)) ([863c88d](https://www.github.com/looker-open-source/sdk-codegen/commit/863c88dfb0569f7c829602acee2404e1c840097e))

### Bug Fixes

* updated TypeScript, C#, Go language names in codeGenerators.ts ([#610](https://www.github.com/looker-open-source/sdk-codegen/issues/610)) ([13fb42b](https://www.github.com/looker-open-source/sdk-codegen/commit/13fb42b6bfa651a137e75b04fc6ee00620705ede))

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @looker/run-it bumped from ^0.9.2 to ^0.9.3
    * @looker/sdk-codegen bumped from ^21.0.12 to ^21.0.13
  * devDependencies
    * @looker/sdk-codegen-scripts bumped from ^21.0.12 to ^21.0.13
    * @looker/sdk-node bumped from ^21.4.1 to ^21.4.2
</details>
<details><summary>@looker/run-it: 0.9.3</summary>

### Features

* introduces prism-editor package; implements read-only usage ([#588](https://www.github.com/looker-open-source/sdk-codegen/issues/588)) ([863c88d](https://www.github.com/looker-open-source/sdk-codegen/commit/863c88dfb0569f7c829602acee2404e1c840097e))

### Bug Fixes

* updated TypeScript, C#, Go language names in codeGenerators.ts ([#610](https://www.github.com/looker-open-source/sdk-codegen/issues/610)) ([13fb42b](https://www.github.com/looker-open-source/sdk-codegen/commit/13fb42b6bfa651a137e75b04fc6ee00620705ede))

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @looker/sdk-codegen bumped from ^21.0.12 to ^21.0.13
</details>
<details><summary>@looker/sdk-codegen-scripts: 21.0.13</summary>

### Bug Fixes

* updated TypeScript, C#, Go language names in codeGenerators.ts ([#610](https://www.github.com/looker-open-source/sdk-codegen/issues/610)) ([13fb42b](https://www.github.com/looker-open-source/sdk-codegen/commit/13fb42b6bfa651a137e75b04fc6ee00620705ede))

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @looker/sdk-codegen bumped from ^21.0.12 to ^21.0.13
    * @looker/sdk-node bumped from ^21.4.1 to ^21.4.2
</details>
<details><summary>@looker/sdk-codegen: 21.0.13</summary>

### Features

* adds 'SDK Examples' data table ([#602](https://www.github.com/looker-open-source/sdk-codegen/issues/602)) ([3678c96](https://www.github.com/looker-open-source/sdk-codegen/commit/3678c96cc7c2edadd00013b66711c917a62d1000))

### Bug Fixes

* updated TypeScript, C#, Go language names in codeGenerators.ts ([#610](https://www.github.com/looker-open-source/sdk-codegen/issues/610)) ([13fb42b](https://www.github.com/looker-open-source/sdk-codegen/commit/13fb42b6bfa651a137e75b04fc6ee00620705ede))
</details>
<details><summary>@looker/extension-sdk-react: 21.4.2</summary>

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @looker/extension-sdk bumped from ^21.4.1 to ^21.4.2
</details>
<details><summary>@looker/wholly-sheet: 0.5.3</summary>

### Dependencies

* The following workspace dependencies were updated
  * devDependencies
    * @looker/sdk-node bumped from ^21.4.1 to ^21.4.2
</details>
<details><summary>@looker/hackathon: 21.4.2</summary>

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @looker/extension-sdk bumped from ^21.4.1 to ^21.4.2
    * @looker/extension-sdk-react bumped from ^21.4.1 to ^21.4.2
    * @looker/wholly-sheet bumped from ^0.5.2 to ^0.5.3
</details>
<details><summary>@looker/extension-api-explorer: 21.4.2</summary>

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * @looker/api-explorer bumped from ^0.9.2 to ^0.9.3
    * @looker/extension-sdk bumped from ^21.4.1 to ^21.4.2
    * @looker/extension-sdk-react bumped from ^21.4.1 to ^21.4.2
    * @looker/run-it bumped from ^0.9.2 to ^0.9.3
    * @looker/sdk-codegen bumped from ^21.0.12 to ^21.0.13
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  "packages/api-explorer": "0.9.2",
-  "packages/extension-api-explorer": "21.4.1",
-  "packages/extension-sdk": "21.4.1",
-  "packages/extension-sdk-react": "21.4.1",
-  "packages/hackathon": "21.4.1",
-  "packages/run-it": "0.9.2",
+  ".": "1.3.0",
+  "packages/api-explorer": "0.9.3",
+  "packages/extension-api-explorer": "21.4.2",
+  "packages/extension-sdk": "21.5.0",
```
Comment (joeldodge79) [inline .release-please-manifest.json:5]: should be 21.4.2

File: .release-please-manifest.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+  "packages/extension-sdk-react": "21.4.2",
+  "packages/hackathon": "21.4.2",
+  "packages/run-it": "0.9.3",
   "packages/sdk": "21.4.1",
-  "packages/sdk-codegen": "21.0.12",
-  "packages/sdk-codegen-scripts": "21.0.12",
+  "packages/sdk-codegen": "21.1.0",
```
Comment (joeldodge79) [inline .release-please-manifest.json:10]: should be 21.0.13

**Review by joeldodge79 — COMMENTED** [review]
my assessment of what versions need to be set with `"release-as"` in release-please-config.json for our next release.

File: .release-please-manifest.json (hunk 1 above)
Comment (bryans99) [reply .release-please-manifest.json:5]: Hm. Would have though extension-sdk was at 21.4.1. Only extension-sdk-react was impacted by yesterdays change

Comment (joeldodge79) [reply .release-please-manifest.json]: correct: extension-sdk was at 21.4.1 and should be showing up as 21.4.2 in this PR which it is now

File: packages/api-explorer/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,30 @@ All notable changes to this project will be documented in this file.
...
+
+* updated TypeScript, C#, Go language names in codeGenerators.ts ([#610](https://www.github.com/looker-open-source/sdk-codegen/issues/610)) ([13fb42b](https://www.github.com/looker-open-source/sdk-codegen/commit/13fb42b6bfa651a137e75b04fc6ee00620705ede))
+
+
+### Dependencies
+
+* The following workspace dependencies were updated
+  * dependencies
+    * @looker/run-it bumped from ^0.9.2 to ^0.9.3
+    * @looker/sdk-codegen bumped from ^21.0.12 to ^21.0.13
```
Comment (bryans99) [inline packages/api-explorer/CHANGELOG.md:27]: Not a big deal but why not to 21.4.2? Are they independent now

**Review by bryans99 — APPROVED** [review]
LGTM

File: packages/api-explorer/CHANGELOG.md (hunk 3 above)
Comment (joeldodge79) [reply packages/api-explorer/CHANGELOG.md:27]: yeah, they're independent now and we'll probably re-name the package to start over at some point.

**Comment by joeldodge79** [comment]
:robot: Release for @looker/extension-api-explorer is at https://github.com/looker-open-source/sdk-codegen/releases/tag/extension-api-explorer-v21.4.2 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/extension-sdk-react is at https://github.com/looker-open-source/sdk-codegen/releases/tag/extension-sdk-react-v21.4.2 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/extension-sdk is at https://github.com/looker-open-source/sdk-codegen/releases/tag/extension-sdk-v21.4.2 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/hackathon is at https://github.com/looker-open-source/sdk-codegen/releases/tag/hackathon-v21.4.2 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/sdk-node is at https://github.com/looker-open-source/sdk-codegen/releases/tag/sdk-node-v21.4.2 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for sdk-codegen-all is at https://github.com/looker-open-source/sdk-codegen/releases/tag/sdk-codegen-all-v1.3.0 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/api-explorer is at https://github.com/looker-open-source/sdk-codegen/releases/tag/api-explorer-v0.9.3 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/run-it is at https://github.com/looker-open-source/sdk-codegen/releases/tag/run-it-v0.9.3 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/sdk-codegen-scripts is at https://github.com/looker-open-source/sdk-codegen/releases/tag/sdk-codegen-scripts-v21.0.13 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/sdk-codegen is at https://github.com/looker-open-source/sdk-codegen/releases/tag/sdk-codegen-v21.0.13 :sunflower:

**Comment by joeldodge79** [comment]
:robot: Release for @looker/wholly-sheet is at https://github.com/looker-open-source/sdk-codegen/releases/tag/wholly-sheet-v0.5.3 :sunflower:

**Pull request merged**